cassandra/src/main/resources/scalate/helpMenu.ssp

```diff
@@ -947,20 +970,35 @@
         <div id="${sharedStatesId}" class="panel-collapse collapse" role="tabpanel">
             <div class="panel-body">
                It is possible to execute many paragraphs in parallel. However, at the back-end side, we’re still using synchronous queries. <em>Asynchronous execution</em> is only possible when it is possible to return a <strong>Future</strong> value in the <strong>InterpreterResult</strong>. It may be an interesting proposal for the <strong>Zeppelin</strong> project.
-               <br/> 
-               Another caveat is that the same <strong>com.datastax.driver.core.Session</strong> object is used for <strong>all</strong> notebooks and paragraphs. Consequently, if you use the <em>USE keyspace name;</em> statement to log into a keyspace, it will change the keyspace for <strong>all current users</strong> of the Cassandra interpreter because we only create 1 <strong>com.datastax.driver.core.Session</strong> object per instance of <strong>Cassandra</strong> interpreter.
-               <br/> 
-               The same remark does apply to the <strong>prepared statement hash map</strong>, it is shared by <strong>all users</strong> using the same instance of <strong>Cassandra</strong> interpreter.
+               <br/><br/>
+               Recently, <strong>Zeppelin</strong> allows you to choose the level of isolation for your interpreters (see
+               <strong><a href="http://zeppelin.incubator.apache.org/docs/0.6.0-incubating-SNAPSHOT/manual/interpreters.html" target="_blank">Interpreter Binding Mode</a></strong> ).
```

Comment:
Hi @doanduyhai :)
Could you change this link to `http://zeppelin.apache.org/docs/0.6.0-incubating-SNAPSHOT/manual/interpreters.html` ?